src/core/bot/message.py

```
@@ -1,226 +1,43 @@
-import asyncio
-import base64
 import random
-import re
-import threading
-import uuid
-from enum import Enum
-from typing import Optional, Union
 
-from botpy import BotAPI
-from botpy.message import Message, GroupMessage, C2CMessage, DirectMessage
+from nonebot.adapters.onebot.v11 import PrivateMessageEvent,GroupMessageEvent
```

Ruff (0.12.2)

`nonebot.adapters.onebot.v11.PrivateMessageEvent` imported but unused

Remove unused import

(F401)